src/main/java/org/fundacionjala/sfdc/core/Env.java

```diff
@@ -1,109 +1,172 @@
 package org.fundacionjala.sfdc.core;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.util.Properties;
 
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-
 /**
  * Class that gets the Environment config.
  */
 public final class Env {
 
-    private static final Logger LOGGER = LogManager.getLogger();
+  private static final Logger LOGGER = LogManager.getLogger();
```

Comment:
@AT-04/at-04 
one question 
are you using 4 spaces or 2 spaces ?

Please let me know which of them you will use.